frontend/package.json

```diff
@@ -314,6 +314,7 @@
     "node": ">=14.x"
   },
   "resolutions": {
+    "hosted-git-info": "^3.0.8",
```

Comment:
It would be good to track this down to the dependency that includes it and bump that if possible instead of using a resolution. It looks like it might be babel?

Reply:
I see that it's from `node-sass` which was [bumped](https://github.com/openshift/console/pull/7365) into `5.0.0` version by you recently.

Reply:
also...
<img width="795" alt="Screenshot 2021-03-29 at 08 05 26" src="https://user-images.githubusercontent.com/1668218/112793392-86a52a80-9065-11eb-8d22-d0c4016a72d0.png">

Reply:
Looks like you're right

```
❯ yarn why hosted-git-info
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "hosted-git-info"...?
[2/4] 🚚  Initialising dependency graph...
warning Resolution field "jquery@3.5.1" is incompatible with requested version "jquery@~3.4.1"
warning Resolution field "jquery@3.5.1" is incompatible with requested version "jquery@~3.4.1"
warning Resolution field "minimist@1.2.5" is incompatible with requested version "minimist@0.0.8"
warning Resolution field "minimist@1.2.5" is incompatible with requested version "minimist@0.0.8"
warning Resolution field "minimist@1.2.5" is incompatible with requested version "minimist@~0.0.1"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "hosted-git-info@2.7.1"
info Reasons this module exists
   - "_project_#normalize-package-data" depends on it
   - Hoisted from "_project_#normalize-package-data#hosted-git-info"
   - Hoisted from "_project_#node-sass#meow#normalize-package-data#hosted-git-info"
   - Hoisted from "_project_#node-sass#meow#read-pkg-up#read-pkg#normalize-package-data#hosted-git-info"
```

Reply:
Hm, it sounds like we should be using dart sass anyway, though. @jhadvig Do you mind opening a story for that?